src/main/resources/edu/cornell/kfs/module/purap/document/datadictionary/CuElectronicInvoiceRejectDocument.xml

```diff
@@ -19,5 +19,14 @@
     <property name="documentClass" value="edu.cornell.kfs.module.purap.document.CuElectronicInvoiceRejectDocument"/>
    <!--   <property name="baseDocumentClass" value="edu.cornell.kfs.module.purap.document.ElectronicInvoiceRejectDocument"/> --> 
  

```

Comment:
Seems reasonable, since this is a small file and not much is being modified anyway. I'll make that change.